static/generate_i18n_resources_list.py

```diff
@@ -30,7 +30,8 @@ def get_translation_info(filepath):
     with open(filepath, 'r', encoding="utf-8") as f:
         content = json.load(f)
         lang_name = content.get("name")
-        return lang_code, lang_name
+        translation_count = len(content)
+        return lang_code, lang_name, translation_count
```

Comment:
Return directly a dict ?
It would avoid error prone `translation_info[1]` access and no need for `make_language_entry`

Reply:
I agree, seems like a now unnecessary intermediate step

Reply:
I used a tuple at this point so that the language list could be sorted (by ISO code) and `language.js` is generated in a well-defined order. However processing of *.json files already takes place in that order, thus the extra sorting doesn't make sense.